meson.build

```diff
@@ -157,42 +148,35 @@ _files = [
     'lib/xmlrole.c',
     'lib/xmltok.c',
   ),
-  config_h,
 ]
 
 # the lib/libexpat.def file is not valid with mingw, only with mvsc, intel-cl,
 # and clang-cl
+extra_kwargs = {}
 if cc.get_id() != 'gcc'
-  libexpat = library(
-    'expat',
-    _files,
-    vs_module_defs : 'lib/libexpat.def',
-    include_directories : include_directories('lib'),
-    dependencies : dep_libbsd,
-    version : '1.6.11',
-    soversion : host_machine.system() != 'windows' ? '1' : '',
-    install : true,
-    gnu_symbol_visibility : 'hidden',
-  )
-else
-  libexpat = library(
-    'expat',
-    _files,
-    include_directories : include_directories('lib'),
-    dependencies : dep_libbsd,
-    version : '1.6.11',
-    soversion : host_machine.system() != 'windows' ? '1' : '',
-    install : true,
-    gnu_symbol_visibility : get_option('build_tests') ? 'default' : 'hidden',
-  )
+  extra_kwargs += { 'vs_module_defs' : 'lib/libexpat.def' }
```

Comment:
there aren't spaces around braces anywhere else in the file.

Reply:
Ok, fixed.